hack/lib/mutil-arch.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Use multiarch/qemu-user-static that is to enable an execution of different multi-architecture containers
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+
+OS=$(awk '/DISTRIB_ID=/' /etc/*-release | sed 's/DISTRIB_ID=//' | tr '[:upper:]' '[:lower:]')
+docker run --rm -t arm64v8/$OS uname -m
+
+# Update buildx plugin to version 6.0+.
+mkdir -pv ~/.docker/cli-plugins/
+wget -O ~/.docker/cli-plugins/docker-buildx \
+    https://github.com/docker/buildx/releases/download/v0.6.3/buildx-v0.6.3.linux-amd64
```

Comment:
please get arch from current build machine first instead of hard `amd64`
```suggestion
    https://github.com/docker/buildx/releases/download/v0.6.3/buildx-v0.6.3.linux-$arch
```

Reply:
fix it